server/models/Notification.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require("mongoose");
+
+const notificationSchema = new mongoose.Schema({
+  // generally, there should be three broad categories of notification: notifications for messages, notifications for payment
+  // activity, notifications for appointment activity and notifications for account activity (login/logout)
+  type: {
+    type: String,
+    enum: ['account', 'appointment', 'message', 'payment'],
+  },
+  description: {
+    type: String,
+    default: "",
+  },
+  read: {
+    type: Boolean,
+    default: false,
+  },
+  dateCreated: {
+    type: Date,
+    default: Date.now,
+  },
+  createdBy: {
+    type: String,
```

Comment:
For `createdBy` and `receivedBy` fields we should reference the appropriate model 😄

Reply:
Changed to referencing the appropriate model in commit.